src/resources/hooks/useFetchNameFromUUID.tsx

```diff
@@ -18,20 +16,33 @@ export const useFetchNameFromUUID = (
 
   const [recipientName, setRecipientName] = useState<string | undefined>(undefined);
   const [error, setError] = useState(false);
+  const [skipUser, setSkipUser] = useState(userUUID === null);
+  const [skipParty, setSkipParty] = useState(partyUUID === null);
```

Comment:
Instead of adding another state that needs to be maintained can't you use the userUUUID to see if you should skipParty?

Reply:
You're right!
I couldn't get it to work without states in the beguinning, but that must have been because of something else because now it suddenly works.
I'll push the change 😊